ui/src/main/webapp/src/app/analysis-context/analysis-context-form.component.html

```diff
@@ -1,5 +1,13 @@
-<div class="container-fluid">
-    <h1 i18n>Configuration</h1>
+<div class="panel-body">
```

Comment:
@klinki - That seems like a reasonable fix, at least until we separate these roles completely.

It seems inevitable that they will be.